packages/backend/src/services/DashboardService/DashboardService.ts

```diff
@@ -154,6 +154,7 @@ export class DashboardService
             loomTilesCount: dashboard.tiles.filter(
                 ({ type }) => type === DashboardTileTypes.LOOM,
             ).length,
+            tabsCount: dashboard.tabs.length || 0,
```

Comment:
`tabs` can be undefined

afaik dashboard.tabs.length will throw an error if tabs is undefined

for this to work we might need todo `dashboard.tabs?.length || 0`
or `(dashboard.tabs || []).length`

Reply:
That's true